src/app/search/search-workflow-table/search-workflow-table.component.scss

```diff
@@ -10,3 +10,12 @@
 .mat-column-projectLinks {
   max-width: 105px;
 }
+
+.language-badge {
```

Comment:
Looks like a duplicate.  Could also use a comment on where the background color and color was gotten from and whether it's something we should be using for all "badges" (not to be confused with Material badges).

Reply:
also why do the tools and workflows have different padding?

Reply:
They look different if you put the exact same padding. Think it might have something to do with the fact that the tool styling is a span inside a div element for the ngFor, but for workflows its in a single span